dev-packages/e2e-tests/test-applications/node-otel-without-tracing/tests/transactions.test.ts

```diff
@@ -34,17 +34,55 @@ test('Sends an API route transaction to OTLP', async ({ baseURL }) => {
   const scopeSpans = json.resourceSpans?.[0]?.scopeSpans;
   expect(scopeSpans).toBeDefined();
 
-  // Http server span & undici client spans are emitted
+  // Http server span & undici client spans are emitted,
+  // as well as the spans emitted via `Sentry.startSpan()`
   // But our default node-fetch spans are not emitted
-  expect(scopeSpans.length).toEqual(2);
+  expect(scopeSpans.length).toEqual(3);
```

Comment:
This was also incorrect before, spans from `startSpan()` did not end up here, although they should. This was probably because of incorrect propagation in this scenario.